tmk_core/common/keycode.h

```diff
@@ -121,6 +121,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define KC_PEQL KC_KP_EQUAL
 #define KC_PCMM KC_KP_COMMA
 
+/* System Specfic */
```

Comment:
```suggestion
/* System specific */
```
Typo again. I still think it makes the most sense to put these between the _Consumer Page (0x0C)_ block and the _Mouse Keys_ block.